package/CHANGELOG

```diff
@@ -48,6 +48,8 @@ Enhancements
    (Issue #4948 #2874, PR #4965)
  * Enables parallelization for analysis.lineardensity.LinearDensity
    (Issue #4678, PR #5007)
+ * Enabled selection of backend for distance calculations in RDF classes
```

Comment:
You did more than just RDF. Also describe separately the changes in `lib.distances`.